packages/playground/src/components/node_selector/TfManualNodeSelector.vue

```diff
@@ -160,6 +170,8 @@ export default {
 
           case props.filters.ipv4 && farms[0].publicIps.every(p => p.contract_id !== 0):
             throw `Node ${nodeId} is not assigned to a PublicIP`;
+          case !features.every(value => node.features.includes(value)):
```

Comment:
![image](https://github.com/user-attachments/assets/db95f37f-7f12-4df8-b031-27466bb0abf1)
![Screenshot from 2024-11-19 16-24-38](https://github.com/user-attachments/assets/e3fa2a27-f56e-43ad-a833-7793f4439c75)

i suggest to show what is the missing feature as in memory, storage etc

Reply:
if it's more than one or two features, the error might be long and won't look good, what do you think?

Reply:
Did we agree to show the missing feature? right ?

Reply:
Yes and it's already implemented